grails-app/assets/javascripts/profileEditor/controllers/AdminController.js

```diff
@@ -225,4 +227,28 @@ profileEditor.controller('ALAAdminController', function ($http, util, messageSer
             self.loadingTags = false;
         });
     }
+
+    function cacheManagement() {
+        self.loadingCacheManagement = true;
+        var promise = $http.get(util.contextRoot() + "/admin/cacheManagement");
+        promise.then(function (response) {
+            self.cacheRegions = response.data || [];
+            self.loadingCacheManagement = false;
+        }, function() {
+            self.loadingCacheManagement = false;
+        });
+    }
+
+    self.clearCache = function (cache) {
+        var promise = $http.get(util.contextRoot() + "/admin/clearCache/" + cache);
+        promise.then(function(response) {
+            if (response.data.error) {
+                messageService.alert(response.data.error);
+            } else {
+                messageService.success(response.data.resp);
+            }
+        }, function() {
+            messageService.alert(response.data.error);
```

Comment:
Function does not have response parameter. Use a custom error message instead of reading message from response.